src/functionalTest/kotlin/com/autonomousapps/fixtures/LeakCanaryProject.kt

```diff
@@ -35,9 +35,25 @@ class LeakCanaryProject(private val agpVersion: String) {
   )
 
   val expectedAdviceForApp = setOf(
+    Advice.ofChange(
+      ModuleCoordinates("com.squareup.leakcanary:leakcanary-android-core", "2.2", GradleVariantIdentification.EMPTY),
+      fromConfiguration = null, toConfiguration = "androidTestImplementation"
+    ),
```

Comment:
You don't want `ofChange`, you want `ofAdd`, which has `fromConfiguration = null` already.